### PR TITLE
Add a statement of vision at softwarejameson.dance/our-vision.html

### PR DESCRIPTION
**Pull request by jamesonwilliams**

### CODE REVIEW

**Review by medelman17 — CHANGES_REQUESTED** [review]
Link to my Twitter, bruh.

**Review by ericlewis — CHANGES_REQUESTED** [review]
I think the clout address might be wrong so..

**Comment by medelman17** [comment]
Also, it would be ideal if you, @jamesonwilliams, could rearticulate the rationale behind your proposed changes into a user story.

File: our-vision.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                 href="https://twitter.com/softwarejameson">softwarejameson</a> kind of looked like this one dude in a
+        TikTok meme. Well? The rest is history. We assembled an
+        engineering team, scrapped together our first prototype and went
+        in for a big round of funding.</p>
+
+        <p>That's where you come in. Have you considered purchasing
+        <a
+            href="https://bitclout.com/u/softwarejameson/buy">softwarejameson
+            coins on BitClout</a>? Think about it. Many good coins are
+        still available.</p>
```
Comment (ericlewis) [inline our-vision.html]:
```suggestion
        <p>That's where you come in. Have you considered donating to
        <a
            href="https://supporters.eff.org/donate">the
            EFF</a>? Think about it. Privacy is at stake! Look! People can just make embarrassing websites about other people!</p>
```

**Review by ericlewis — CHANGES_REQUESTED** [review]

**Review by ericlewis — APPROVED** [review]

**Review by medelman17 — APPROVED** [review]

**Comment by medelman17** [comment]
DUDE! Squash merge?? Hello!? FML

**Pull request merged**